code/go/internal/validator/folder_item_content.go

```diff
@@ -17,8 +17,8 @@ func loadItemContent(itemPath, mediaType string) ([]byte, error) {
 		return nil, errors.Wrap(err, "reading item file failed")
 	}
 
-	if len(itemData) == 0 {
-		return nil, errors.New("file is empty")
+	if len(itemData) == 0 && mediaType != "" {
```

Comment:
It took me a second to realize that this additional check was required to properly handle the `.empty` file case. In the case of the `.empty` file, `len(itemData) == 0` but `mediaType == ""` so this check will evaluate to `false`, which is what we want. Without the additional `mediaType != ""` clause, this check would have evaluated to `true` and returned an error, which is not what we want.

Would you mind adding a comment above this check explaining why we need to check both the length of `itemData` and the media type?

Reply:
Fixed.

Reply:
I'll merge it once the CI is green and doesn't complain.